share/site/duckduckgo/about.tx

```diff
@@ -95,11 +95,11 @@
 		<div class="flex-row">
 			<div class="links__card">
 				<noscript>
-					<img src="/assets/common/icons/privacy-tips-2.svg" aria-hidden="true" class="links__card__icon">
+					<img src="/assets/common/icons/devices.svg" aria-hidden="true" class="links__card__icon">
 				</noscript>
-				<img src="" data-src="/assets/common/icons/privacy-tips-2.svg" aria-hidden="true" class="links__card__icon js-lazysvg">
+				<img src="" data-src="/assets/common/icons/devices.svg" aria-hidden="true" class="links__card__icon js-lazysvg">
 				<h2 class="links__card__title">Privacy, simplified.</h2>
-				<p class="links__card__text">We don’t track you, and we stop others from tracking you, so you can Internet with peace of mind.</p>
+				<p class="links__card__text">Our app and browser extension stop you from being tracked, so you can Internet with peace of mind.</p>
```

Comment:
Not a show stopper: Can we have these in translation tokens?

Reply:
I think that should probably be part of a larger effort to translate all the content on these new pages?

AFAIK nothing else is translated for most (all?) of our static pages.